src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocHtmlFormat.java

```diff
@@ -0,0 +1,51 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+/**
+ * Config: default.
+ */
+public class InputSummaryJavadocHtmlFormat {
+
+    /**
+     * <p><b>{@summary Normal Javadoc.}</b></p>
+     */
+    private void foo1() {} // ok
+
+    /**
+     * <p><b><i>{@summary Normal Javadoc}</i></b></p>
+     */
+    private void foo2() {} // violation
+
+    /**
+     * <p>{@summary Normal Javadoc {@author my code}.}</p>
+     */
+    private void foo3() {} // violation
```

Comment:
Why is this a violation? It looks like this file might be out of date with the test.

Reply:
I missed updating this example, others were updated. Done.